EIPS/eip-epbs.md

```diff
@@ -0,0 +1,260 @@
+
+## Motivation
+
+This EIP solves a different set of unrelated important problems. 
+
+- An overwhelming majority of beacon block proposers outsource the construction of the execution payload within their blocks to a third party (henceforth called a *builder*). In order to do so, they request the hash tree root (HTR) of a promised execution payload and submit a `SignedBlindedBeaconBlock` to a trusted party that is tasked with replacing the HTR with the full execution payload (received from the builder) before broadcasting. This EIP allows for a trust-free direct communication between the beacon block proposer and the builder, guaranteeing that an honest beacon block proposer will receive payment from the builder regardless of the latter actions and that the honest builder's payload will be the canonical head of the chain regardless of the proposer's action. 
+- Currently, validators have the time between receiving the full beacon block (including an execution payload) and the attesting deadline (4 seconds in Ethereum mainnet) to perform both consensus and execution state transition functions, check blob data availability and evaluate the new head of the blockchain. The remainder of the slot time is spent during less CPU-intense and critical task. By separating the validation of the execution and consensus part of the block, validators are only tasked to perform the consensus state transition function in this critical time before attesting, while execution and data availability validation is deferred for most of the remainder of the slot (between the builder's reveal time and the next attestation deadline). 
+- By removing the full execution payload size from the consensus block, it allows for faster network propagation on the critical path. 
+- It removes the increased reorg likeliness of including blob transactions in blocks. 
+- It prevents validators from missing attestations and strengthens the LMD weight properties of forkchoice in the event that builders produce invalid payloads. 
+- It removes the need of using middleware, not maintained by core developers, used by the vast majority of validators and it removes the need from client teams to continue supporting such middleware, improving thus coordination efforts for testing and deploying future hard forks. 
```

Comment:
"the need to use", but I disagree with this formulation. There is no need-to-use middleware today. Validators can local-build. What it does is remove the need to use (trusted) middleware *in the event that the validator chooses to not self-build*. I am also unclear whether the part about "the need for client teams to continue supporting" is relevant here, this feels like it has to do with the meta of protocol R&D, not with protocol specifications per se (also it might be wrong, if client teams need to continue supporting e.g., builder connections)

Reply:
I can remove the sentence, but validators will definitely support the builder API, just that there is no more need for a middle ware. Validators directly contact builders with the builder API. I can either remove the sentence altogether or add an explicit wording for the need if you want to offload building. Either way the claim about clients needing to support and test a middle ware is true regardless

Reply:
So I think they are two separate points, would do two bullets:
- ePBS removes the need to use trusted middleware in order to delegate block construction to a builder.
- ePBS removes the responsibility of testing said middleware from the client teams, improving ...

I am still unclear whether the second bullet is really relevant to a technical specification, and I could see a world where the question whether MEV-Boost should continue to be tested by client teams is debated.

Reply:
Lol my original PR had only the first bullet point. The second one was a request :)